mkdocs.yml

```diff
@@ -32,5 +32,164 @@ extra_javascript:
+          - User personas guide: 110-ux/services/research/user-personas-guide.md
+      - Engineering:
+          - Engineering calls: 060-engineering/README.md
+          - Accessibility: 060-engineering/accessibility.md
+          - Backend development: 060-engineering/back-end-development.md
+          - Development and hosting environments: 060-engineering/dev-environments.md
+          - Drupal - contributed vs. custom development: 060-engineering/most-important-decision-in-developing-a-drupal-site-contributed-vs-custom-development.md
+          - Drupal developer tips for getting the most out of open source: 060-engineering/drupal-developer-tips-for-getting-the-most-out-of-open-source.md
+          - Drupal practice area: 050-how-we-work/practice-areas/drupal-practice-area.md
+          - Drupal site building: 060-engineering/drupal-site-building.md
+          - Engineer's role in client/team relationships: 060-engineering/engineer-role-in-client-relationships.md
+          - Engineering: 060-engineering/README.md
+          - Engineering project roles: 060-engineering/engineering-roles.md
+          - Git: 060-engineering/git.md
+          - How to log internal meetings: 060-engineering/internalmeetings.md
+          - Quality assurance: 060-engineering/qa.md
+          - Security and compliance: 060-engineering/security-compliance.md
+          - Systems and sandboxes: 060-engineering/systems-sandboxes.md
+          - Tech Lead (TL) project role description: 060-engineering/tech-lead.md
+          - CSS (and SASS etc.): 060-engineering/front-end/css.md
+          - Development tools: 060-engineering/front-end/front-end-tools.md
+          - Drupal front-end development: 060-engineering/front-end/drupal-frontend.md
+          - Front-end development: 060-engineering/front-end/README.md
+          - Front-end testing: 060-engineering/front-end/testing.md
+      - Help desk:
+          - Help desk support: 120-help-desk/helpdesk.md
+          - Project support and Agile: 120-help-desk/help-desk-agile.md
+          - Help desk working practices: 120-help-desk/working-practices.md
+      - Project management:
+          - Project manager role: 070-project-management/pm-role.md
+          - Guidelines for starting a new project: 070-project-management/starting-new-project.md
+          - Invoicing: 070-project-management/invoicing.md
+          - Onboarding a new project team member: 070-project-management/onboarding-new-project-team-member.md
+          - When your project gets a contractual Modification (MOD) or Extension: 070-project-management/mods-and-extentions.md
+          - Tracking contract expirations and renewals: 070-project-management/contract-expiration-tracking.md
+          - Tracking contractual requirements: 070-project-management/contractual-requirements.md
+          - Conflict resolution and growth mindset: 070-project-management/growth-mindset/#what-is-growth-mindset
```

Comment:
@ContentBrewmaster this link is going to the URL instead of the `.md` version of the file.